cp23/hpi-fun2/motivation/problem.json

```diff
@@ -0,0 +1,9 @@
+{
+  "description": "traverse hierarchy to find matching candidates",
```

Comment:
The description should not contain story elements as this should be something that describes your problem in a concise formal way. Something like:
```suggestion
  "description": "Minimum limited path cover on a vertex-weighted directed tree",
```